packages/language/src/index.js

```diff
@@ -13,36 +13,46 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/*!
- * @module language
- * @name Language
```

Comment:
@lukesneeringer just like https://github.com/GoogleCloudPlatform/google-cloud-node/pull/2613#discussion_r139265635 these are required in order to build the docs.

Reply:
Thank you!